docs/upgrade-guides/upgrading-nextjs-from-v4-to-v5.mdx

```diff
@@ -25,191 +25,126 @@ To aid you in this upgrade, Clerk built a CLI called [`@clerk/upgrade`](https://
+    ```
 
-      ```bash filename="terminal"
-      yarn add react@latest react-dom@latest
-      ```
+    ```bash filename="terminal"
+    yarn add react@latest react-dom@latest
+    ```
 
-      ```bash filename="terminal"
-      pnpm add react@latest react-dom@latest
-      ```
-    </CodeBlockTabs>
+    ```bash filename="terminal"
+    pnpm add react@latest react-dom@latest
+    ```
+</CodeBlockTabs>
 
-1. Update Next.js
+If you are upgrading from react 17 or lower, make sure to [learn about how to upgrade your react version to 18](https://react.dev/blog/2022/03/08/react-18-upgrade-guide) as well.
 
-    You will need to update your Next.js project to 13.0.4 or later. Check out Next's upgrade guides:
+#### Updating Next.js
 
-    - [Upgrading from 12 to 13](https://nextjs.org/docs/pages/building-your-application/upgrading/version-13)
-    - [Upgrading from 13 to 14](https://nextjs.org/docs/app/building-your-application/upgrading/version-14)
+`@clerk/nextjs` now requires you to use Next.js version `13.0.4` or later. Check out Next's upgrade guides for more guidance if you have not yet upgraded to next 13:
 
-## Redesigned components
+- [Upgrading from 12 to 13](https://nextjs.org/docs/pages/building-your-application/upgrading/version-13)
+- [Upgrading from 13 to 14](https://nextjs.org/docs/app/building-your-application/upgrading/version-14)
 
-v5 ships with improved design and UX on all of Clerk's [UI components](/docs/components/overview). If you have used the [`appearance` prop](/docs/components/customization/overview) or tokens for a [custom theme](/docs/components/customization/overview#create-a-custom-theme), you will likely need to make some adjustments to ensure your styling is still looking great. If you're using the [localization prop](/docs/components/customization/localization) you will likely need to make adjustments to account for added or removed localization keys.
+## New middleware architecture
 
-<Callout type="info" >
-  Use the `@clerk/upgrade` CLI to more quickly find changes required for the redesigned components.
-</Callout>
+We have been actively fielding user and customer feedback about `authMiddleware()` and have heard from many that this was a friction point. As such, in v5 we are releasing a completely new middleware helper called [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware) that we think will alleviate all of the issues folks had with `authMiddleware()`.
 
-Follow the instructions below to update your `appearance` prop and localization keys to the new structure.
+Additionally, we have added new helpers that enable controlling auth on a _per-page and/or per-layout basis_. While you can still configure which routes are protected via middleware, the page-based protection strategy is what we now recommend, as we feel the co-location of auth protection configuration with the code for the pages themselves results in a superior developer experience that adds clarity as you navigate your codebase.
```

Comment:
https://github.com/clerk/clerk-docs/blob/main/STYLEGUIDE.md#21-you-is-the-reader-clerk-is-clerk